src/worker.js

```diff
@@ -24,6 +24,7 @@ const state = {
   vehicles: {},
   springs: {},
   springInstances: {},
+  constraints: {},
```

Comment:
I named this `constraints` similar to how the `postStep` callback was stored in `springs`: https://github.com/pmndrs/use-cannon/blob/master/src/worker.js#L389-L391

Ideally I think these all can be renamed to more informative / accurate variable names. It works as is though!